### PR TITLE
mark errormessage strings for translation

### LINKED ISSUE #3066: untranslated API messages
### Top-level intent

What did you try to achieve?

Display localized error messages from privacyIDEA REST API in simpleSAMLphp. I guess there are two approaches, either do the translation on privacyIDEA server or ignore messages from server and use translated strings based on error codes.

### Steps to reproduce

1. use [simpleSAMLphp module privacyIDEA](https://github.com/privacyidea/simplesamlphp-module-privacyidea)
2. enter (for example) a wrong TOTP
3. an error message is shown, like `"Response did not match for 5 tokens"` ([source](https://github.com/privacyidea/privacyidea/blob/4ea557be5004feeed957fbee6cd78c1d417cc75f/privacyidea/lib/token.py#L2376))

### Expected outcome

The error message should be in the user's language (if available).

### Actual outcome

The error message is always in English, because the error messages are not being translated (in source code).

### Configuration

* **privacyIDEA version**: 3.7-dev4
* **Installation method**: github
* **Python version**: 3.9
* **Operating system**: debian
* **Webserver**: uwsgi
* **Token database**: PostgreSQL

### PR DESCRIPTION
**Pull request by Sebastian-Lemling**
closes #3066

### CODE REVIEW

File: privacyidea/lib/token.py [hunk 1]
```diff
@@ -561,9 +561,9 @@ def get_one_token(*args, **kwargs):
     """
     result = get_tokens(*args, **kwargs)
     if not result:
-        raise ResourceNotFoundError(u"The requested token could not be found.")
+        raise ResourceNotFoundError(_(u"The requested token could not be found."))
```
Comment (jona-samuel) [inline privacyidea/lib/token.py:564]: I think the 'u' is redundant, as the default string type is Unicode. In line 566 too

**Review by jona-samuel — CHANGES_REQUESTED** [review]
There are translations missing in lines 1334 and 1873

File: privacyidea/lib/token.py (hunk 1 above)
Comment (NNLemling) [reply privacyidea/lib/token.py:564]: after consultation it was decided to leave the 'u' standing.

**Review by jona-samuel — APPROVED** [review]
Lucks good

**Pull request merged**